transport/src/main/java/io/el/channel/Channel.java

```diff
@@ -39,6 +39,8 @@ public interface Channel {
 
   ChannelPromise connect(SocketAddress remoteAddress);
 
+  ChannelPromise connect(SocketAddress remoteAddtess, SocketAddress localAddress);
```

Comment:
I think when connecting local address would not need

Reply:
I removed it!